### PR TITLE
feat(frontend): SideBar's Footer

### PR DESCRIPTION
**Pull request by lfaucon**
* Remove privacy policy and Navigation on HomePage

* Add a Footer in the sidebar with useful links

Screenshot 2021-11-20 at 10 15 41

### CODE REVIEW

File: frontend/src/features/frame/components/sidebar/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+
+import { makeStyles } from '@material-ui/core/styles';
+
+import { handleWikiUrl } from 'src/utils/url';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    padding: 8,
+    borderTop: '1px solid #806300',
```
Comment (GresilleSiffle) [inline frontend/src/features/frame/components/sidebar/Footer.tsx:10]: personal preference

The color `#806300` is quite strong, I would have used the color `#e7e5db` suggested by Figma instead.

File: frontend/src/features/frame/components/sidebar/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    display: 'flex',
+    flexFlow: 'row wrap',
+    justifyContent: 'space-between',
+  },
+  menuLink: {
```
Comment (GresilleSiffle) [inline frontend/src/features/frame/components/sidebar/Footer.tsx:15]: How about adding a `textDecoration: 'underline'` on hover?

The majority of our clickable elements in the UI have a little effect when the mouse is over, and a little visual hint here might help to navigate in the footer.

**Review by GresilleSiffle — CHANGES_REQUESTED** [review]
The new footer looks great!

I checked the **request changes** checkbox, but one of them is an aesthetic personal preference so feel free to ignore it : )

File: frontend/src/features/frame/components/sidebar/Footer.tsx (hunk 2 above)
Comment (lfaucon) [reply frontend/src/features/frame/components/sidebar/Footer.tsx:15]: Improved!

File: frontend/src/features/frame/components/sidebar/Footer.tsx (hunk 1 above)
Comment (lfaucon) [reply frontend/src/features/frame/components/sidebar/Footer.tsx]: Improved!

**Comment by lfaucon** [comment]
Thanks for the suggestions @GresilleSiffle 👍

**Review by GresilleSiffle — APPROVED** [review]
Perfecto!

**Pull request merged**